js/jobmixins.js

```diff
@@ -354,13 +358,14 @@ exports.do_download = function(op, callback) {
  * encounter a collision.
  */
 var saveToDeviceStorage = exports.saveToDeviceStorage =
-function(_LOG, blob, storeTo, filename, partInfo, cb, isRetry) {
+function(_LOG, blob, storeTo, registerDownload, filename, partInfo, cb,
+         isRetry) {
```

Comment:
pop3/sync.js seems to reference this function, looks like it needs a signature change?